packages/xforms-engine/src/instance/input/InitialInstanceState.ts

```diff
@@ -44,12 +62,35 @@ const parseInstanceDocument = (model: ModelDefinition, instanceXML: string): Sta
 export class InitialInstanceState {
 	static async from(
 		model: ModelDefinition,
-		sources: InitialInstanceStateSources
+		data: InitialInstanceStateSources
 	): Promise<InitialInstanceState> {
-		const [instanceData] = sources;
-		const instanceFile = instanceData.get(INSTANCE_FILE_NAME);
-		const instanceXML = await getBlobText(instanceFile);
-		const attachments = new InstanceAttachmentMap(sources);
+		return this.resolve(model, {
+			inputType: 'FORM_INSTANCE_INPUT_RESOLVED',
```

Comment:
Question:

I've been wondering for a while. What are the criteria to make all uppercase vs lowercase? For example: `ready` and `FORM_INSTANCE_INPUT_RESOLVED`? Just curious, no asking to change anything

Reply:
Before vs after [we decided on this convention](https://github.com/getodk/web-forms/pull/334#discussion_r1994304892) (also captured in #338).

Reply:
Oh yeah got it, this is the new way, and the lower-case is the old way